maven-plugin/src/test/resources-its/org/jvnet/localizer/LocalizerPluginIT/smokeTest/src/test/java/org/jvnet/localizer/test/MessagesTest.java

```diff
@@ -0,0 +1,13 @@
+package org.jvnet.localizer.test;
+
+import org.junit.Test;
+
+import org.junit.Assert;
+
+public class MessagesTest {
+
+    @Test
+    public void smokeTest() throws Exception {
```

Comment:
the old test project had no tests and needed to be manually verified.
this adds a very simple test to that project so that the IT will fail